src/sphinxcontrib/bibtex/__init__.py

```diff
@@ -43,16 +47,30 @@ def setup(app: Sphinx) -> Dict[str, Any]:
     app.add_directive("bibliography", BibliographyDirective)
     app.add_role("cite", CiteRole())
     app.add_node(bibliography, override=True)
-    app.add_node(raw_latex, latex=(visit_raw_latex, depart_raw_latex),
-                 override=True)
+    app.add_node(raw_latex, latex=(visit_raw_latex, depart_raw_latex), override=True)
     app.add_post_transform(BibliographyTransform)
     app.add_domain(BibtexFootDomain)
     app.add_directive("footbibliography", FootBibliographyDirective)
     app.add_role("footcite", FootCiteRole())
 
+    # Catch bug in newer docutils
+    if (
+        (0, 18, 0, "final", 0, True)
+        <= docutils.__version_info__
+        < (0, 20, 0, "final", 0, True)
+    ) and app.builder.name == "html":
+        error = RuntimeError(
+            (
+                "Beware that docutils versions 0.18 and 0.19 (you are running {}) are known to generate invalid html "
+                "for citations. If this issue affects you, please use docutils<=0.17 (or >=0.20 once released) instead."
+                " For more details, see https://sourceforge.net/p/docutils/patches/195/"
+            ).format(docutils.__version__)
+        )
+        SPHINX_LOGGER.warn(error.args[0])
```

Comment:
Why not just emit the warning message directly (instead of first wrapping it inside a runtime error)?

Reply:
Also the SPHINX_LOGGER variable appears nowhere to be defined and/or imported.